src/components/progress-summary/progress-extractor.service.ts

```diff
@@ -12,7 +12,11 @@ export class ProgressExtractor {
     @Logger('progress:extractor') private readonly logger: ILogger
   ) {}
 
-  extractCumulative(pnp: WorkBook, file: FileVersion): Progress | null {
+  extractCumulative(
+    pnp: WorkBook,
+    file: FileVersion,
+    fiscalYear: number
+  ): Progress | null {
     // new standard 2020 version has new sheet "Harvest" which isolates relevant progress data
     const sheet = pnp.Sheets.Harvest ?? pnp.Sheets.Progress;
```

Comment:
We shouldn't need to worry about the Harvest tab either. It doesn't have the data we are looking for.

Reply:
It doesn't? Isn't `cumulative` the same as `summary` was before?

Reply:
Oh right, because you're wanting to grab the longer decimal values now.